cluster-stamp.json

```diff
@@ -98,36 +98,6 @@
                 "description": "AKS Service, Node Pool, and supporting services (KeyVault, App Gateway, etc) region. This needs to be the same region as the vnet provided in these parameters."
             }
         },
-        "geoRedundancyLocation": {
```

Comment:
please let's rebase on top of the latest changes.

Reply:
done